.gitignore

```diff
@@ -4,3 +4,4 @@
 /composer.lock
 /vendor
 /phar/solarium.phar
+.idea
```

Comment:
@mkalkbrenner this should be (ideally) defined as a common ignore pattern on developer's side ... WDYT?